config/locales/pl.yml

```diff
@@ -80,9 +80,9 @@ pl:
       show_places: 'Pokaż miejsca'
       new_event: 'Dodaj nowe wydarzenie'
       you_are_member: 'Jesteś uczestnikiem'
+      incomming: 'Nadchodzący'
```

Comment:
In this situation translation will be visible only on `index.html` page. In everything else we will get missing translations.